mautic/mautic
# Fix audit log for few entities

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
This is follow up https://github.com/mautic/mautic/pull/7311
Prevent made audit log without change



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Edit email - change from name, from address, reply to and bcc  and save
3. Got to audit_log table - after PR you see this changelog in audit_log
4. Edit Form - do not any change - and save form
5. Go to audit_log - you see change noIndex change even If no change chapend - after PR no audit log without change
6. Edit Page - do not any change - and save page - you see change noIndex and isPreferencesCenter change even If no change chapend - after PR no audit log without change
7. Edit Segment - do not any change - and save segment - you see change isGlobal and isPreferencesCenter change even If no change chapend - after PR no audit log without change
